src/commands/test.js

```diff
@@ -13,8 +13,7 @@ const test = (context) => {
     extraEnv.DETAILED_LOG_TO_STDOUT = 'true';
   }
 
-  if (process.version !== LAMBDA_VERSION) {
-    context.line(`You're running tests on Node ${process.version}, but Zapier runs your code on ${LAMBDA_VERSION}. The version numbers must match. See https://zapier.github.io/zapier-platform-cli/index.html#requirements for more info.`);
+  if (!utils.correctVersion()) {
```

Comment:
ohhh you're super right. I thought the question is "is it really necessary to pass context into this function at all". I'll fix!